src/components/LoadingScreen.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState, useEffect } from "react";
```

Comment:
The component currently does not include ARIA attributes or roles. Adding attributes like role="progressbar", aria-valuenow, aria-valuemin, and aria-valuemax would make the component more accessible, particularly for screen reader users.